From 4d2c2370dc85e658508fd0943e47b0d6559f42d5 Mon Sep 17 00:00:00 2001
From: Michael Simons <msimons@microsoft.com>
Date: Mon, 9 Aug 2021 14:03:33 +0000
Subject: [PATCH] NoWarn NU1605 until DeploymentTools is in SourceBuild

src/Cli/dotnet/dotnet.csproj : error NU1605: Detected package downgrade: Newtonsoft.Json from 12.0.3 to 12.0.2. Reference the package directly from the project to select a different version.

Error stems from Microsoft.Deployment.DotNet.Releases not being source built and referencing a newer version.  Temporary fix until the DeploymentTools repo is added to source-build (https://github.com/dotnet/source-build/issues/2365).
---
 src/Cli/dotnet/dotnet.csproj | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/Cli/dotnet/dotnet.csproj b/src/Cli/dotnet/dotnet.csproj
index 4ff965a4d..f9609824b 100644
--- a/src/Cli/dotnet/dotnet.csproj
+++ b/src/Cli/dotnet/dotnet.csproj
@@ -12,6 +12,7 @@
     <PackageId>Microsoft.DotNet.Cli</PackageId>
     <UseAppHost>false</UseAppHost>
     <IsPackable>true</IsPackable>
+    <NoWarn>$(NoWarn);NU1605</NoWarn>
   </PropertyGroup>
   <ItemGroup>
     <Compile Remove="commands\dotnet-new\**" />
-- 
2.29.2

